src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.ServerCertificates.cs

```diff
@@ -152,5 +152,12 @@ public static IEnumerable<object[]> UseCallback_ValidCertificate_ExpectedValuesD
                 Assert.True(callbackCalled);
             }
         }
+
+        private static bool BackendSupportsCustomCertificateHandling =>
+            RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ||
+            (CurlSslVersionDescription()?.StartsWith("OpenSSL") ?? false);
```

Comment:
Is this safe?  I don't know how late the JIT guarantees extern method binding.  If it's "not until first called", then yay.